pkg/eventcollector/eventcollector.go

```diff
@@ -142,7 +142,7 @@ func (c *EventCollector) handleEvent(payload interface{}) interface{} {
 		eventType, txHash.Hex(), hash) // Debug, remove later
 
 	if c.crawlerPubSub != nil {
-		err = c.crawlerPubSub.PublishWatchedEventMessage(event)
+		err = c.crawlerPubSub.PublishTriggerMessage()
```

Comment:
Well actually,the newsroom events that need to be handled will be handled with that exceptionmessage that's sent

Reply:
Just going to keep that trigger message there because in the case where there may be lots of newsroom events that need to be added, at least the most recent tcr event will be processed first